packages/embark/src/lib/modules/ethereum-blockchain-client/index.js

```diff
@@ -7,11 +7,12 @@ import checkContractSize from "./checkContractSize";
 
 class EthereumBlockchainClient {
 
-  constructor(embark, options) {
+  constructor(embark, _options) {
```

Comment:
Just a tiny remark (and surely a matter of taste as well):

In cases like this where there's more parameters available than actually needed, I'd just remove them from the signature. Even though the `_` indicates that we aren't interested in it, it still makes me wonder every time for a little moment, where that parameter is used.

If we aren't interested in a parameter and all of its (possible) following parameters aren't used either, we can simply leave it out.